rosidl_runtime_c/include/rosidl_runtime_c/bounds_type_support_struct.h

```diff
@@ -23,25 +23,25 @@ extern "C"
 {
 #endif
 
-typedef struct rosidl_message_bounds_t rosidl_message_bounds_t;
+typedef struct rosidl_bounds_type_support_t rosidl_bounds_type_support_t;
 
-typedef const rosidl_message_bounds_t * (* rosidl_message_bounds_handle_function)(
-  const rosidl_message_bounds_t *, const char *);
+typedef const rosidl_bounds_type_support_t * (* rosidl_message_bounds_handle_function)(
+  const rosidl_bounds_type_support_t *, const char *);
 
-struct rosidl_message_bounds_t
+struct rosidl_bounds_type_support_t
```

Comment:
While we are in here, we may as well document what this structure is meant to do, and what the fields are.  I know we are missing it from the other "type support" structures, but you gotta start somewhere :).

Reply:
I started here to document these headers and structs https://github.com/ros2/rosidl/pull/472 . @mjcarroll , it would be greate if you can review it .

Reply:
I'll review, thanks for that @ahcorde